File: 백준/Bronze/2475. 검증수/검증수.java
import java.util.*;
import java.io.*;

public class Main {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int sum = 0;
        br.close();

        for(int i = 0; i < 5; i++) {
            int num = Integer.parseInt(st.nextToken());
            sum += (int) Math.pow(num, 2);
        }

        bw.write(String.valueOf(sum % 10));
        bw.flush();
        bw.close();
    }
}

File: 백준/Bronze/2475. 검증수/검증수.kt
import java.io.*
import java.util.*
import kotlin.math.pow

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.out))

    val st = StringTokenizer(br.readLine())
    br.close()

    var sum = 0.0
    repeat(5) {
        sum += st.nextToken().toDouble().pow(2)
    }

    bw.write((sum % 10).toInt().toString())
    bw.flush()
    bw.close()
}